### PR TITLE
Jrb more cleanup

### PR DESCRIPTION
**Pull request by jayrbolton**
This is just a syntax cleanup.
- Move things into a conventional python style. Instead of having `index.py` in the root, have a `src/server.py` file. The python package is all under `src`. 
- Use a lint-fix tool to automatically get index.py to pass flake8 linter

### CODE REVIEW

File: src/static/nav-highlight.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        .querySelector('.table tbody tr:nth-child(' + currentElem + ') a')
+        .removeAttribute('style');
+  }
+  document.body
+      .querySelector('.table tbody tr:nth-child(' + elemIndex + ') a')
+      .setAttribute('style', 'color: #576523; font-weight: bold;');
+  currentElem = elemIndex;
+};
+
+// Run the callback `fn` at most every `ms`
```
Comment (FlyingUnicornSF) [inline src/static/nav-highlight.js]: would you also change the comment?

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[flake8]
```
Comment (FlyingUnicornSF) [inline tox.ini]: is this file usually in repo?

File: src/static/nav-highlight.js (hunk 1 above)
Comment (jayrbolton) [reply src/static/nav-highlight.js]: 👍 donezo

File: tox.ini (hunk 2 above)
Comment (jayrbolton) [reply tox.ini]: Yes, most often

**Comment by jayrbolton** [comment]
Okay restored config.ini

**Pull request merged**